src/style.css

```
@@ -0,0 +1,128 @@
+/* #region body */
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  color: #060b35;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+}
+/* #endregion */
+
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 32px 16px 16px;
+  max-width: 200px;
+  box-sizing: border-box;
```

```suggestion
  * {box-sizing: border-box;}
```
це можна одразу до всього прописати